packages/xstate-wallet/src/workflows/application.ts

```diff
@@ -131,8 +131,12 @@ const signFinalState = (store: Store) => async ({channelId}: ChannelIdExists) =>
     .toPromise();
   // Sign a finalized state
   const {supported} = ourTurnEntry;
-  const finalState = {...supported, turnNum: supported.turnNum.add(1), isFinal: true};
-  return store.signAndAddState(channelId, finalState);
+  const finalState = {
+    outcome: supported.outcome as SimpleAllocation,
```

Comment:
Might need some runtime check here. Or, we could disable `MixedAllocation` at the type-level for now since we don't support it.